app/models/legal_aid_application.rb

```diff
@@ -132,6 +132,14 @@ def set_transaction_period
     )
   end
 
+  def uncategorised_income_transactions?
+    @bank_transactions = bank_transactions.credit.order(happened_at: :desc).by_type
+    transaction_types.credits.each do |transaction_type|
```

Comment:
Why the instance variable here?  It isn't used outside this method is it?  Would prefer to see a local variable called `grouped_transactions` or something similar.